### DISCUSSION TITLE
No full information with all options ON

### DISCUSSION THREAD

**Discussion by PinKevin** [community/community#133386] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

At past two weeks, I experienced this issue. My dashboard doesn't display all of the options that I selected. Please give some solution, because my dashboard experience got horrible with this issue

**Comment by LiteBrite82** [comment]
💬 Your Product Feedback Has Been Submitted 🎉
Thank you for taking the time to share your insights with us! Your feedback is invaluable as we build a better GitHub experience for all our users.

Here's what you can expect moving forward ⏩
- Your input will be carefully reviewed and cataloged by members of our product teams. 
- Other users may engage with your post, sharing their own perspectives or experiences. 
- GitHub staff may reach out for further clarification or insight. 

Where to look to see what's shipping 👀
- Read the [Changelog](https://github.blog/changelog/) for real-time updates on the latest GitHub features, enhancements, and calls for feedback.
- Explore our [Product Roadmap](https://github.com/orgs/github/projects/4247), which details upcoming major releases and initiatives.

What you can do in the meantime 💻
- Upvote and comment on other user feedback Discussions that resonate with you.
- Add more information at any point! Useful details include: use cases, relevant labels, desired outcomes, and any accompanying screenshots.

As a member of the GitHub community, your participation is essential. While we can't promise that every suggestion will be implemented, we want to emphasize that your feedback is instrumental in guiding our decisions and priorities.

Thank you once again for your contribution to making GitHub even better! We're grateful for your ongoing support and collaboration in shaping the future of our platform. ⭐
